source/create.ts

```diff
@@ -241,6 +241,11 @@ const create = (defaults: InstanceDefaults): Got => {
 			if (optionsToMerge !== undefined) {
 				normalizedOptions = normalizeArguments(undefined, optionsToMerge, normalizedOptions);
 			}
+
+			// eslint-disable-next-line no-await-in-loop
+			await new Promise(resolve => {
```

Comment:
This should be a utility method in https://github.com/sindresorhus/got/tree/master/source/utils

Reply:
Done